bin/ensembl_database_map.txt

```diff
@@ -0,0 +1,18 @@
+species	command
+cel	useMart(biomart = "ensembl", dataset = "celegans_gene_ensembl", host="https://www.ensembl.org", archive=FALSE)
```

Comment:
This fetches the newest version every time, right? I am a bit worried about reproducibility and compatibility in the long run. Would there be a way to cache it?